trie.py

```diff
@@ -0,0 +1,73 @@
+# https://www.codingninjas.com/studio/problems/implement-trie_631356?leftPanelTabValue=PROBLEM
+from sys import stdin, setrecursionlimit
+setrecursionlimit(10**7)
+
+
+class Node:
+    def __init__(self):
+        self.links = [None] * 26
+        self.flag = False
```

Comment:
Flag is a generic term, what it means here?

Reply:
Flag ending indicates whether the current char is the word ending or not

Reply:
Then isn't `end_of_word` or `end` is better indicator?

Reply:
Done